### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by lperanni**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](/help/leaving-feedback-in-github)”.
</details>

### CODE REVIEW

File: form.html [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+                <input id="email" type="email" placeholder=" Enter your Email" required>
+            </label>
+            
+            <label id="number-label">Age (optional)
+                <input id="number" type="number" placeholder=" Age" name="number-label" min="10" max="99">
+            </label>
+    
+            <label for="current-role">Which option best describes your current role?</label>
+            <select id="current-role" name="current-role" required>
+                <option value="">Select current role</option>
```
Comment (Volki312) [inline form.html:32]: <option value="" **selected="selected" disabled**>Select current role</option>

File: form.html [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+                <li> 
+                    <label for="not-sure">
+                        <input id="not-sure" type="radio" value="not-sure" name="recommendation"> Not sure
+                    </label>
+                </li>
+            </ul>
+
+            <label for="dropdown">What is your favourite feature of freeCodeCamp?</label>
+            <select id="dropdown" name="dropdown" required>
+                <option value="">Select an option</option>
```
Comment (Volki312) [inline form.html:61]: <option value="" **selected="selected" disabled**>Select an option</option>

File: form.html [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+                <input id="name" type="text" placeholder=" Enter your name" required>
+            </label>
+
+            <label id="email-label">Email
+                <input id="email" type="email" placeholder=" Enter your Email" required>
+            </label>
+            
+            <label id="number-label">Age (optional)
+                <input id="number" type="number" placeholder=" Age" name="number-label" min="10" max="99">
+            </label>
```
Comment (Volki312) [inline form.html:28]: label elements missing **for** attributes

File: style.css [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    font-family: 'Roboto', monospace, sans-serif;
+    font-size: 20px;
+    color: white;
+    background-color: #494691;
+}
+
+header {
+    margin: 10%;
+    text-align: center;
+    font-size: 1rem;
```
Comment (Volki312) [inline style.css:11]: While em is relative to the font-size of its direct or nearest parent, rem is only relative to the html (root) font-size.

rem is mostly used for styles such as button paddings, while em is most suitable for font-size.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    background-color: #37af65;
+    color: white;
+    border-color: #37af65;
+    display: block;
+    width: 100%;
+    border-radius: 2px;
+    margin-top: 20px;
+    margin-bottom: 30px;
+    height: 40px;
+}
```
Comment (Volki312) [inline style.css:55]: Use [border property](https://www.w3schools.com/css/css_border.asp) **(border: none;)** to remove the border from html elements

File: form.html (hunk 3 above)
Comment (nparunov) [reply form.html]: Added

File: style.css (hunk 4 above)
Comment (nparunov) [reply style.css]: Changed

File: style.css (hunk 5 above)
Comment (nparunov) [reply style.css]: Added

**Pull request merged**